clients/web/src/views/widgets/HierarchyWidget.js

```diff
@@ -681,6 +707,21 @@ girder.views.HierarchyWidget = girder.View.extend({
         }).on('g:saved', function (folder) {
             // need to do anything?
         }, this);
+    },
+
+    /**
+     * Select (highlight) an item in the list.
+     * @param item An ItemModel instance representing the item to select.
+     */
+    selectItem: function (item) {
```

Comment:
I forgot that was in there, thanks for reminding me. I will add a test for this.